inspirehep/modules/theme/static/scss/format/detailed-record.scss

```diff
@@ -222,7 +221,12 @@ $darken: 8%;
 	display: none;
 }
 
-#record-abstract-title, #record-keywords-title, #record-reference-title, #record-citation-title, #record-impact-graph {
+#record-abstract-title,
+#record-keywords-title,
+#record-reference-title,
+#record-citation-title,
+#record-impact-graph,
+#record-conferences-title {
```

Comment:
This selector feels weird. Maybe a class would be better, since 6 IDs share the same styling?